include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -764,13 +775,14 @@ class __future : private std::tuple<_Args...>
 #endif
     }
 
+    template <std::size_t _Idx = 0>
     auto
-    get()
+    get(std::size_t __idx = 0)
```

Comment:
Everybody who will take a look at this function will ask - what is template param and run-time param here,
Explanation required, I think.

Reply:
Better to understand this from code...